### ISSUE TITLE
OK to gitignore Assets/Plugins/Dissonance/Resources/.UpdateState.json file?

### ISSUE THREAD

**Issue by haydenjameslee** [Placeholder-Software/Dissonance#86]
After updating to Dissonance 6.1 we're seeing `Assets/Plugins/Dissonance/Resources/.UpdateState.json` change quite frequently. Is it ok to git ignore this file?

**Comment by martindevans** [comment]
Gitignoring that should be fine. That file suppresses the update notifier from showing up more than once per version - so you'll probably get a notification about an update once-per-clone-of-the-project instead of once-per-project.

However, it shouldn't be changing frequently. What values are you seeing appear in the file?

**Comment by martindevans** [comment]
I did some investigation into this and have just submitted a PR with some improvements. I'm assuming the changes you're seeing in the file are to the `_nextCheckFileTime`, this updates once every 12-72 hours when it performs a check for a new version of Dissonance. I've moved the contents of that file into the `PlayerPrefs` so they're not in a file and won't cause this kind of confusion in the future (as well as a few other minor improvements).
